modules/client-configuration/pages/clients-rh-cdn.adoc

```diff
@@ -255,7 +255,7 @@ The channels you need for this procedure are:
 |===
 
 
-include::snippets/add_channels_cli.adoc[]
+include::snippets/add_channels_scc.adoc[]
```

Comment:
SCC is not used for Red Hat itself, only for the client tools.

Reply:
Yes, this is for client tools under Uyuni. Client tools under SUMA uses the WebUI.